src/components/Chart/Chart.tsx

```diff
@@ -2,12 +2,19 @@ import React from "react";
 import "./Chart.css";
 import ChartBar from "./ChartBar";
 
-const Chart = (props) => {
-  const numValues = props.dataValue.map((expense) => expense.value);
+type labelFields = {
+  label: string;
+  value: number;
+};
+
+const Chart = (props: { dataValue: labelFields[] }) => {
+  const numValues = props.dataValue.map(
+    (expense: labelFields) => expense.value
+  );
   const totalMax = Math.max(...numValues);
   return (
     <div className="chart">
-      {props.dataValue.map((dataValue) => {
+      {props.dataValue.map((dataValue: labelFields) => {
```

Comment:
As you have properly typed the `dataValue` prop, setting it for the map variable isn't needed. TS's inference is good enough to figure out this type on it's own.
However, it's better to differentiate the actual array, over which you map, with the singular element, that is provided by the `.map()` method. This way the code is easier to understand.
Ex.: `dataValue` -> `dataValues` and then when you map over it, it becomes `dataValues.map((datavalue))`